x-pack/plugins/security_solution/server/lib/telemetry/endpoint_metadata_telemetry.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Client, ApiResponse } from '@elastic/elasticsearch';
+import { SearchResponse } from 'elasticsearch';
+import { HostMetadata, HostPolicyResponse } from '../../../common/endpoint/types';
+import { getESQueryPolicyResponseByHostID } from '../../endpoint/routes/policy/service';
+
+export interface EndpointMetadataOSTelemtry {
```

Comment:
:grey_question: sp `Telemtry`

Reply:
Let the games begin:

| Speller | Score |
| ---- | ---- |
| @michaelolo24 | 1 |
| @bkimmel  | 1 |

Reply:
fixed 😄